scripts/test/Muon/elemental_analysis_2/EA_plotting_widget/EA_plot_data_pane_model_test.py

```diff
@@ -0,0 +1,101 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+from unittest import mock
+from Muon.GUI.ElementalAnalysis2.ea_group import EAGroup
+from Muon.GUI.ElementalAnalysis2.plotting_widget.EA_plotting_pane.EA_plot_data_pane_model import EAPlotDataPaneModel
+from mantidqt.utils.qt.testing import start_qapplication
+from Muon.GUI.Common.test_helpers.context_setup import setup_context_for_ea_tests
+
+
+@start_qapplication
+class EAPlotDataPaneModelTest(unittest.TestCase):
+
+    def setUp(self):
+        setup_context_for_ea_tests(self)
+        self.model = EAPlotDataPaneModel(context=self.context)
+        test_group = EAGroup(group_name="9999; Detector 1", detector="Detector 1", run_number="9999")
+        self.context.group_context._groups = [test_group]
+        self.context.group_context._selected_groups = ["9999; Detector 1"]
+
+    def test_get_workspace_to_plot_counts(self):
+        self.context.group_context["9999; Detector 1"].get_counts_workspace_for_run = mock.MagicMock()
+
+        self.model.get_count_workspaces_to_plot("9999; Detector 1", True)
+
+        self.context.group_context["9999; Detector 1"].get_counts_workspace_for_run.assert_called_once()
```

Comment:
you can check the arguments and that things you dont want to be called are not called (e.g. get_rebined_or_unbinned_version_of_workspace_if_it_exists)